examples/langmuir_turbulence.jl

```diff
@@ -82,20 +82,19 @@ Qᵘ = -3.72e-5 # m² s⁻², surface kinematic momentum flux
 
 u_boundary_conditions = UVelocityBoundaryConditions(grid, top = BoundaryCondition(Flux, Qᵘ))
 
-# On buoyancy, the initial condition and bottom boundary condition impose the
-# linear buoyancy gradient `N²`. McWilliams et al. (1997) also impose a weak,
-# destabilizing flux of buoyancy at the surface to avoid spurious laminarization of the
-# near-surface velocity field.
+# McWilliams et al. (1997) impose a linear buoyancy gradient `N²` at the bottom
+# along with a weak, destabilizing flux of buoyancy at the surface to faciliate
+# spin-up from rest.
 
 Qᵇ = 2.307e-9 # m³ s⁻², surface buoyancy flux
 N² = 1.936e-5 # s⁻², initial and bottom buoyancy gradient
 
 b_boundary_conditions = TracerBoundaryConditions(grid, top = BoundaryCondition(Flux, Qᵇ),
                                                        bottom = BoundaryCondition(Gradient, N²))
 
-# Note that Oceananigans uses "positive upward" conventions for all fluxes. In consequence,
-# a negative flux at the surface drives positive velocities, and a positive flux of
-# buoyancy drives cooling.
+# !!! Note that Oceananigans uses "positive upward" conventions for all fluxes. In consequence,
```

Comment:
Is this supposed to be a docs admonition? You missed something...

Reply:
Ah yes thanks for catching